tests/plugins/test_resolve_composes.py

```diff
@@ -1561,7 +1561,7 @@ def test_canceling_compose_when_timeout_of_waiting_for_the_compose(
         # Ensure ODCSClient.wait_for_compose raises timeout error
         (flexmock(time)
          .should_receive('time')
-         .and_return(2 + ODCSClient.DEFAULT_WAIT_TIMEOUT, 1)
```

Comment:
Why this change?  If it's unrelated fix it should be at least in separate commit

Reply:
It was detected during running tests with the new code changes. I'm actually confused why this change is needed. But, it just works now. :)